### PR TITLE
Pathfinder Support Added

### PR DESCRIPTION
**Pull request by talktocory**
@JochenLinnemann ,

I've added support for Pathfinder to the importer.  As discussed, the approach was changed to modifying the HeroLab generated html instead of attempting to parse the character information from the xml.  It's possible that this change means that the add-on can support all of the game systems that HeroLab supports so long as the .por file creation process is the same for all of them.

Cory

### CODE REVIEW

File: src/CampaignKit.PortfolioImporter/Entities/Character.cs [hunk 1]
```diff
@@ -45,8 +47,22 @@ public class Character
...
-    }
+		#endregion
+
+		#region Abstract Members
+
+		/// <summary>
+		/// Return the character formatted in the system default format.
+		/// </summary>
+		/// <returns></returns>
+		public abstract string getDefaultFormat();
```
Comment (JochenLinnemann) [inline src/CampaignKit.PortfolioImporter/Entities/Character.cs]: Please use PascalCasing for Namespaces, Types, Properties, Methods, and camelCasing for parameters and fields.

Comment (JochenLinnemann) [reply src/CampaignKit.PortfolioImporter/Entities/Character.cs]: Nevermind, I can do the formatting afterwards.

File: src/CampaignKit.PortfolioImporter/Entities/StatBlock5e.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace CampaignKit.PortfolioImporter.Entities
+{
+	[XmlRoot(ElementName = "creature-heading")]
+    public class Creatureheading
```
[the remark this answers is not in the document]
Comment (JochenLinnemann) [reply src/CampaignKit.PortfolioImporter/Entities/StatBlock5e.cs]: Same here, I will ReSharper run through this.

**Comment by talktocory** [comment]
Great!

If you look in the entities folder you'll see the full XML model for
HeroLab 5e and Pathfinder.  It was a HUGE pain to put that together.  I had
to keep making new .por files to get samples of all the various XML
variations.  I know we're not using this right now but it may come into
play later so I thought I'd include it.

**Comment by JochenLinnemann** [comment]
We'll keep these files, for sure!

**Comment by JochenLinnemann** [comment]
New version is online and working nicely! Thanks for all your work!

**Pull request merged**